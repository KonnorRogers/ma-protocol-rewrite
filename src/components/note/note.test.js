import React from "react"
import { render } from "@testing-library/react"
import Note from "./note.js"

test("Renders without error", () => {
  const tree = render(<Note />)

  expect(tree).toMatchSnapshot()
})
