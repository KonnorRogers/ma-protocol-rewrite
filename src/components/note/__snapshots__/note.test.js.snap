// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without error 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  margin: 0.75rem;
  padding: 1.5rem 2rem;
  border-radius: 12px;
  border: 2px solid #6666ff;
  text-align: center;
  font-size: 1rem;
}

@media only screen and (min-width:640px) {
  .emotion-0 {
    font-size: 1.1rem;
  }
}

.emotion-0 {
  margin: 0.75rem;
  padding: 1.5rem 2rem;
  border-radius: 12px;
  border: 2px solid #6666ff;
  text-align: center;
  font-size: 1rem;
}

@media only screen and (min-width:640px) {
  .emotion-0 {
    font-size: 1.1rem;
  }
}

<body>
    <div>
      <div
        class="emotion-0"
      />
    </div>
  </body>,
  "container": .emotion-0 {
  margin: 0.75rem;
  padding: 1.5rem 2rem;
  border-radius: 12px;
  border: 2px solid #6666ff;
  text-align: center;
  font-size: 1rem;
}

@media only screen and (min-width:640px) {
  .emotion-0 {
    font-size: 1.1rem;
  }
}

.emotion-0 {
  margin: 0.75rem;
  padding: 1.5rem 2rem;
  border-radius: 12px;
  border: 2px solid #6666ff;
  text-align: center;
  font-size: 1rem;
}

@media only screen and (min-width:640px) {
  .emotion-0 {
    font-size: 1.1rem;
  }
}

<div>
    <div
      class="emotion-0"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
